### ISSUE TITLE
"GPU process isn't usable."

### ISSUE THREAD

**Issue by garaboncias** [wireapp/wire-desktop#5257]
Wire version: 3.26.2941
Operating system: Ubuntu 21-10 (Not released yet)
Which antivirus software do you have installed: There is no antivir

What steps will reproduce the problem?

1.  start wire-desktop

What is the expected result?
Start the application

What is the actual result?
Crash

Please provide any additional information below. Attach a screenshot if possible.
```
(node:32620) UnhandledPromiseRejectionWarning: Error: ERR_INVALID_URL (-300) loading ''
    at rejectAndCleanup (electron/js2c/browser_init.js:205:1493)
    at Object.failListener (electron/js2c/browser_init.js:205:1706)
    at Object.emit (events.js:315:20)
    at NavigationController.reload (electron/js2c/browser_init.js:205:2748)
    at BrowserWindow.n.reload (electron/js2c/browser_init.js:33:2136)
    at App.<anonymous> (/opt/Wire/resources/app.asar/electron/dist/main.js:267:18)
    at App.emit (events.js:315:20)
(node:32620) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:32620) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
[32620:0816/142646.708261:FATAL:gpu_data_manager_impl_private.cc(445)] GPU process isn't usable. Goodbye.
fish: “wire-desktop” terminated by signal SIGTRAP (Trace or breakpoint trap)
```

**Comment by garaboncias** [comment]
Starting with `wire-desktop --no-sandbox` works

**Comment by spaghetus** [comment]
Is there any known fix for this yet? I don't use Wire personally but this issue affects *every* form of Chromium on my system, including the standalone browser, CEF, and everything based on Electron. I'm using OpenSUSE Tumbleweed.

**Comment by garaboncias** [comment]
Used these apps with Xorg (instead of wayland) or with no-sandbox flag is the only workaround what I know.

However desktop screen share also not works with wayland.

**Comment by RDKRACZ** [comment]
bump

**Comment by DemiMarie** [comment]
Does `--disable-gpu` help?

**Comment by plumchii** [comment]
> Does `--disable-gpu` help?
Using `--disable-gpu` caused the same crash. This is using the Appimage on X11.

**Comment by jknockel** [comment]
The issue may be related to the one described here:

https://bugs.launchpad.net/ubuntu/+source/glibc/+bug/1944468/comments/9

If so, the issue appears fixed in chromium as of June 2021:

https://chromium.googlesource.com/chromium/src/+/218438259dd795456f0a48f67cbe5b4e520db88b

From what I can tell, Wire uses version 87.0.4280.141 of chromium, which is from January 2021.

I wonder if there are also security implications to using a browser engine that is this old.

**Comment by DemiMarie** [comment]
> I wonder if there are also security implications to using a browser engine that is this old.

There are.

**Comment by phpwutz** [comment]
welcome to 2022 one of the biggest social network owners :)
whatsapp web still does'nt work on latest arch linux in 2022 for exactly the same error. 
There's no friggin reason whatsapp functionality wise that i see that you cannot support the web version without extra GPU. 
I understand you cannot support any legacy old stuff and the service hasn't been working for *month*, but i'm on the latest version of chrom/ium/FF but at this point i feel a big resentment for a couple month. 
If a company doesn't deem users of up2d8 browsers worth using their service, i'll just switch to recommend to my recommend to my 200 private trustees using matrix. the idea is better anyway. thanks for giving me a good reason.
Love to the developers, disappointment to the ppl making stupid long term decisions. 
good job everyone 💩

**Comment by flokli** [comment]
It seems this got fixed in the 3.27.2944 (linux) release, at least I don't need to set `--no-sandbox` anymore.
